codes/javascript/chapter_computational_complexity/leetcode_two_sum.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * File: leetcode_two_sum.js
+ * Created Time: 2022-12-15
+ * Author: gyt95 (gytkwan@gmail.com)
+ */
+
+/**
+ * @param {number[]} nums
+ * @param {number} target
+ * @return {number[]}
+ */
+function twoSumBruteForce(nums, target) {
+    let n = nums.length;
+    // 两层循环，时间复杂度 O(n^2)
+    for (let i = 0; i < n; i++) {
+        for (let j = i + 1; j < n; j++) {
+            if (nums[i] + nums[j] === target) {
+                return [i, j];
+            }
+        }
+    }
```

Comment:
```suggestion
    }
    return [];
```
You need to consider when we can not find the result.
We should follow the Java code to translate.